### PR TITLE
[REF] mass_editing: migrate module to version 10.0

### PR DESCRIPTION
**Pull request by zaoral**
I made the minimum changes in order to run this module in 10.0.
Update README and descriptions of the repository and the module it self
to refer this module as a 10.0 module y not a 9.0 unported module.

### CODE REVIEW

**Comment by zaoral** [comment]
@nhomar here is the PR you requested
cc @moylop260

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ addon | version | summary
 [date_range](date_range/) | 10.0.1.0.0 | Manage all kind of date range
 [dbfilter_from_header](dbfilter_from_header/) | 10.0.1.0.0 | Filter databases with HTTP headers
 [disable_odoo_online](disable_odoo_online/) | 10.0.1.0.0 | Remove odoo.com Bindings
+[mass_editing](mass_editing/) | 10.0.1.0.0 | Mass Editing
```
Comment (moylop260) [inline README.md]: This change is automatically executed then you don't need added it here

**Review by moylop260 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zaoral) [reply README.md]: Got it, thank you @moylop260 
I just made a new push removing this change.

Best Regards

**Comment by mourad-ehm** [comment]
Code review :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot :+1:

**Comment by bealdav** [comment]
LGTM :+1:

**Comment by moylop260** [comment]
@pedrobaeza 
The migration is fine?

**Comment by pedrobaeza** [comment]
Why are you duplicating another PR for the same module that is in https://github.com/OCA/server-tools/pull/598?

**Comment by pedrobaeza** [comment]
Closing this one as it's very uncompleted and merging the right one.

**Pull request closed without merging**